homeassistant/components/heos/services.yaml

```diff
@@ -1,3 +1,17 @@
+group_volume_set:
+  target:
+    entity:
+      integration: heos
+      domain: media_player
+  fields:
+    volume_level:
+      required: true
+      selector:
+        number:
+          min: 0
+          max: 1
+          step: 0.01
```

Comment:
Can we do everything on 0-100? I think that's more logical

Reply:
0-1 in the scale we use for ’volume_level’, so I didn't want to make this different--that would be confusing if ’group_volume_level’ is then 1-100.

Reply:
oh right, that is true